src/main/kotlin/no/nav/amt/arena/acl/processors/SakProcessor.kt

```diff
@@ -0,0 +1,32 @@
+package no.nav.amt.arena.acl.processors
+
+import no.nav.amt.arena.acl.domain.kafka.arena.ArenaSakKafkaMessage
+import no.nav.amt.arena.acl.exceptions.IgnoredException
+import no.nav.amt.arena.acl.repositories.ArenaSakRepository
+import org.springframework.stereotype.Component
+
+@Component
+open class SakProcessor(
+	private val arenaSakRepository: ArenaSakRepository
+) : ArenaMessageProcessor<ArenaSakKafkaMessage> {
+
+	companion object {
+		private const val SAKSKODE_TILTAK = "TILT"
+	}
+
+	override fun handleArenaMessage(message: ArenaSakKafkaMessage) {
+		val sak = message.getData().mapSak()
+
+		if (sak.sakskode != SAKSKODE_TILTAK) {
+			throw IgnoredException("Sak med kode ${sak.sakskode} er ikke relevant")
+		}
+
+		arenaSakRepository.upsertSak(
```

Comment:
Her ville jeg lagt gjennomføringen på kafka. Tror det ville vært like enkelt å konstruere gjennomføringen på nytt fra arena data isteden for å scripte i databasen i prod. Da har vi også koden som dokumentasjon på historikk og det er ikke sikkert man trenger å gjøre noe mer med det senere.